src/SFA.DAS.Funding.ApprenticeshipEarnings.DataTransferObjects/Learner.cs

```diff
@@ -4,17 +4,19 @@ namespace SFA.DAS.Funding.ApprenticeshipEarnings.DataTransferObjects
 {
     public class Learner
     {
-        public Learner(string uln, FundingType fundingType, List<OnProgrammeEarning> onProgrammeEarnings, decimal totalOnProgrammeEarnings)
+        public Learner(string uln, FundingType fundingType, List<OnProgrammeEarning> onProgrammeEarnings, decimal totalOnProgrammeEarnings, bool isNoneLevyFullyFunded)
         {
             Uln = uln;
             FundingType = fundingType;
             OnProgrammeEarnings = onProgrammeEarnings;
             TotalOnProgrammeEarnings = totalOnProgrammeEarnings;
+            IsNoneLevyFullyFunded = isNoneLevyFullyFunded;
         }
 
         public string Uln { get; set; }
         public FundingType FundingType { get; set; }
         public List<OnProgrammeEarning> OnProgrammeEarnings { get; set; }
         public decimal TotalOnProgrammeEarnings { get; set; }
+        public bool IsNoneLevyFullyFunded { get; set; }
```

Comment:
Approving as this is very minor but there is a bit of a mixture of NoneLevy and NonLevy variable names throughout